libft/libft/ft_split.c

```diff
@@ -6,7 +6,7 @@
 /*   By: yoav <yoav@student.42.fr>                  +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2022/06/14 15:29:04 by yoav              #+#    #+#             */
-/*   Updated: 2022/06/15 13:51:05 by yoav             ###   ########.fr       */
+/*   Updated: 2022/09/17 13:41:05 by alopez-g         ###   ########.fr       */
```

Comment:
it is better to not push changes like this since they are redundent and make it harder to do CR